NAPS2.Core/Automation/AutomatedScanning.cs

```diff
@@ -421,7 +419,7 @@ private bool IsPdfFile(string path)
         {
             string extension = Path.GetExtension(path);
             Debug.Assert(extension != null);
-            return extension.ToLower() == ".pdf";
+            return string.Equals(extension, ".pdf", StringComparison.CurrentCultureIgnoreCase);
```

Comment:
What's in it?

Reply:
@timvdh Did you have a chance to see my other reply, or to try out a build of this? It's basically code refactoring against newer .NET Framework.